### ISSUE TITLE
In the lecture "A Problem that Stumped Milton Friedman", the code need to be sophisticated

### ISSUE THREAD

**Issue by Yuya-Furusawa** [QuantEcon/lecture-source-py#305]
When I tried to simulate the model with the following initial distribution, these are very similar distributions,  I got the unexpected results.

<img width="959" alt="Distribution" src="https://user-images.githubusercontent.com/35675799/58422967-e8458800-80ce-11e9-92a9-332e84c150e1.png">

In this simulation, the cutoff values are
α = 0.48241206030150763
β = 0.5226130653266331
However, from the model, we must have α > β. We should check whether α is larger than β, or not.

**Comment by jstac** [comment]
@AnjuJoon would you mind to look at this, and consult with @shizejin if you're not sure?

**Comment by AnjuJoon** [comment]
@jstac yeah sure, I will look into that

**Comment by jstac** [comment]
@shizejin If you have time would you mind to review this and give me your opinion?

**Comment by shizejin** [comment]
@jstac Sure, here is what I think and I made a PR to solve this.

@Yuya-Furusawa Thank you very much for pointing out this problem.

α and β can be equal, which is the case under the parameterization you gave. Intuitively speaking, when f0 and f1 are too similar and hard to be distinguished from each other, the gains from taking draws would be too small that we will just choose f0 or f1 by priori.

The current code fails to find out this equality. The mathematical definition of α and β is

<img width="946" alt="Screen Shot 2019-06-19 at 17 28 02" src="https://user-images.githubusercontent.com/22466278/59753954-c9927580-92b7-11e9-859a-45b405bd10cf.png">

while the code is not consistent with the math, and consequently the computation result is incorrect.

<img width="768" alt="Screen Shot 2019-06-19 at 15 57 35" src="https://user-images.githubusercontent.com/22466278/59754349-7ec52d80-92b8-11e9-8ff9-50c4cee73bd9.png">

I've made a PR #435 to solve this problem, and the result seems correct.

<img width="771" alt="Screen Shot 2019-06-19 at 17 08 11" src="https://user-images.githubusercontent.com/22466278/59754314-6c4af400-92b8-11e9-8f3a-da4090afd00d.png">
